roles/content_view_publish/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# defaults file for content_view_publish
+FOREMAN_VALIDATE_CERTS: true
```

Comment:
nothing uses this variable, the playbook uses `foreman_validate_certs` as it is defined in the "common" variables